.gitignore

```diff
@@ -0,0 +1,27 @@
+keras_cv.egg-info/
+*.swp
+.idea
+*.pyc
+.pytest_cache
+*.egg-info
+
+#VS Code files and container
+.vscode/
+.devcontainer/
+
```

Comment:
Basically everything below this line can be deleted

Reply:
done